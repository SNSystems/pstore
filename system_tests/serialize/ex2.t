# %binaries = the directories containing the executable binaries
# %T = the test output directory
# %S = the test source directory
REQUIRES: examples
RUN: %binaries/example_write_pod_struct > %T/ex2_actual.txt
RUN: diff %T/ex2_actual.txt %S/ex2_expected.txt
