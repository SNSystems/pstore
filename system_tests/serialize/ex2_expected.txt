Wrote these bytes:
1e 00 00 00 28 00 00 00
